framework/src/main/java/org/tron/core/db/Manager.java

```diff
@@ -1597,6 +1534,10 @@ public BlockCapsule generateBlock(Miner miner, long blockTime, long timeout) {
         postponedTrxCount++;
         continue; // try pack more small trx
       }
+      // check transaction
+      if (!transactionValidator.silentValidate(trx)) {
```

Comment:
Actually, some checks may be skipped if that has been checked when it was pushed into pending queue.

Reply:
Not necessary for now.